test/napi/lib/types.js

```diff
@@ -70,6 +70,15 @@ describe('type checks', function() {
     assert(!addon.is_string(new String('1')));
   });
 
+  it('is_symbol', function () {
+    assert(addon.is_symbol(Symbol()));
+    assert(addon.is_symbol(Symbol("unique symbol")));
+    assert(addon.is_symbol(Symbol.for('neon:description')));
+    assert(addon.is_symbol(Symbol.toStringTag));
```

Comment:
Can you add a test for a built in symbol like `Symbol.iterator`?

Reply:
[Symbol.toStringTag](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toStringTag) is one of the well-knowns, but I'm happy to swap it for one that is more obvious at-a-glance.

Reply:
Sorry just missed this!